README.md

```diff
@@ -889,6 +891,53 @@ If these failures occur, add the warning to the list of permitted warnings found
 in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
 
 
+## macOS Big Sur: pyenv build failed
+
+[pyenv Big Sur GitHub issue](https://github.com/pyenv/pyenv/issues/1740)
+
+macOS Big Sur users may encounter a `build failed` error when installing python through pyenv. 
+To remedy this there is a patch file which runs the installer compatible with 
+macOS Big Sur (11.x).
+
+First is to install zlib and bzip2 using brew:
+```
+brew install zlib bzip2
+```
+
+`brew` output should tell you the steps to editing the path variables LDFLAGS and
+CPPFLAGS.
+The command can be placed in the `~/.bash_profile` to make the change persistent
+
+The command to install python 3.8.3 using the patch file is as follows:
+```
+pyenv install --patch 3.8.3 < <(curl -sSL https://github.com/python/cpython/commit/8ea6353.patch)
+```
+
+
+## macOS Big Sur: Replace preinstall LibreSSL
+
+If the cryptography package fails to install through `make requirements`,
+install OpenSSL through brew (`brew install openssl`) and use the instructions from brew output to
+edit the path variable.
+The above command can be placed in the `~/.bash_profile` to make the change persistent
+
+The above lines can be used in tandem with the LDFLAGS and CPPFLAGS from
+installing zlib and bzip2.
+
+
+## macOS Big Sur: `pip install grpcio` fails due to wheels build file not being found
+
+[Resolution source](https://stackoverflow.com/a/63972598)
+[macOS Big Sur Release Notes](https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11_0_1-release-notes#Third-Party-Apps)
+
+To enable successful installation of the `grpcio` python library, a backwards compatibility
+flag needs to be added to the `~/.bash_profile`:
```

Comment:
This should not be done in the ~/.bash_profile because that can lead to installation issues with other software that this person uses, instead it should be in the environment.local that's located in the project root. This will only get sourced when someone is working with our project.